### PR TITLE
Feat/update to testcafe 1.1.0

### PR DESCRIPTION
**Pull request by matthew-bettingcloud**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gherkin-testcafe",
-  "version": "2.0.0",
+  "version": "3.0.0",
```
Comment (Lukas-Kullmann) [inline package.json]: There is no breaking change. Use v2.1.0.

**Comment by Lukas-Kullmann** [comment]
@matthew-bettingcloud if you want to, feel free to add yourself as a contributor. :)

File: package.json (hunk 1 above)
Comment (matthew-bettingcloud) [reply package.json]: Ok, since TestCafe introduced a breaking change for the package. Updated as requested. Thanks.

**Comment by Lukas-Kullmann** [comment]
Thanks for your contribution. I published v2.1.0 to npm. :)

**Pull request merged**